src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2452,8 +2471,14 @@ subroutine CNOffsetLitterfall (num_soilp, filter_soilp, &
 
             if (offset_counter(p) == dt) then
                t1 = 1.0_r8 / dt
-               leafc_to_litter(p)  = t1 * leafc(p)  + cpool_to_leafc(p)
                frootc_to_litter(p) = t1 * frootc(p) + cpool_to_frootc(p)
+               
+               ! Cut a certain fraction (i.e., harvfrac(ivt(p))) (e.g., harvfrac(ivt(p)=70% for bioenergy crops) of leaf C
+               ! and move this fration of leaf C to biofuel C, rather than move it to litter
+               leafc_to_litter(p)  = t1 * leafc(p)*(1._r8-harvfrac(ivt(p)))  + cpool_to_leafc(p)
+               leafc_to_biofuelc(p) = t1 * leafc(p) * harvfrac(ivt(p))
+               leafn_to_biofueln(p) = t1 * leafn(p) * harvfrac(ivt(p))
```

Comment:
Although it shouldn't matter, I think it would be more consistent with the other code (especially given some of my other comments) if these two lines were moved down into the `if (ivt(p) >= npcropmin)` conditional, making it clear that these fluxes are only set for prognostic crops.

You can then add a comment above pointing out that harvfrac is only non-zero for prognostic crops.